ledger-api/grpc-definitions/com/daml/ledger/api/v1/commands.proto

```diff
@@ -71,6 +74,23 @@ message Commands {
   // Must not be set at the same time as min_ledger_time_abs.
   // Optional
   google.protobuf.Duration min_ledger_time_rel = 11;
+
+  // Set of parties on whose behalf the command should be executed. It is up to the server to verify that the
```

Comment:
I wouldn't phrase it as `It is up to the server to verify` but rather along the lines of "with authorization enabled, the authorization metadata must include all parties provided in ``act_as``". It's also not so much that the connection itself must be authorized, but the individual request needs that authorization.

Reply:
This was copy-🍝  from the existing comment of the `party` field. I'll try to improve both of them.